packages/notte-agent/src/notte_agent/main.py

```
@@ -64,24 +64,32 @@ def create_agent(
             agent = NotifierAgent(agent, notifier=self.notifier)
         return agent
 
-    async def arun(self, task: str, url: str | None = None) -> AgentResponse:
+    async def arun(self, **kwargs: Unpack[AgentRunRequestDict]) -> AgentResponse:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)